src/test/kotlin/day04/WizardTest.kt

```
@@ -0,0 +1,33 @@
+    lateinit var hero: Hero
+    lateinit var wizard: Wizard
+
+    @Before
+    fun setUp() {
+        hero = Hero("hero",5, Sword("common sword", 5))
+        wizard = Wizard("wizard", 10, 5)
+    }
+
+    @After
+    fun tearDown() {
+    }
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)